src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -279,6 +279,7 @@ private CookieOptions BuildCookieOptions()
             var context = new CookieSigningOutContext(
                 Context,
                 Options,
+                new AuthenticationProperties(signOutContext.Properties),
```

Comment:
I'm not sure that's a valid argument right now, since we don't test any of these contexts anyways :)  I'd say making them not settable at all, is actually less surface area in the long term anyways...